proto/cosmos/base/abci/v1beta1/abci.proto

```diff
@@ -52,7 +53,7 @@ message TxResponse {
 message ABCIMessageLog {
   option (gogoproto.stringer) = true;
 
-  uint32 msg_index = 1 [(gogoproto.jsontag) = "msg_index"];
+  uint32 msg_index = 1 [(gogoproto.jsontag) = "msg_index", (cosmos.msg.v1.legacy_amino_dont_omitempty) = true];
```

Comment:
This creates an import cycle:
- the file is generated into `types/abci.go`
- but there's a `types` <-> `types/msgservice` dep cycle

I propose to just remove this annotation TBH, I don't see this proto message used by Amino ever.

Reply:
Probably makes sense to remove